deployment/terraform/cluster/local/apps/variables.tf

```diff
@@ -206,18 +206,78 @@ variable "metrictank" {
   }
 }
 
+#alerting config
+variable "alerting" {
+  type = "map"
+  default = {
+    version = "latest"
+  }
+}
 
-# ========================================
-# Adaptive Alerting
-# ========================================
+# metric-router config
+variable "metric-router" {
+  type = "map"
+  default = {
+    enabled = false
+    metric_router_instances = 1
+    metric_router_cpu_request = "100m"
+    metric_router_cpu_limit = "1000m"
+    metric_router_memory_request = "250"
+    metric_router_memory_limit = "250"
+    metric_router_jvm_memory_limit = "200"
+    metric_router_environment_overrides = ""
+  }
+}
 
-variable "alerting" {
+# metric-router config
+variable "ewma-detector" {
   type = "map"
   default = {
-    version = "02da63dfd5a559c9048b6e83d205d49da3af25f2"
+    enabled = false
+    ewma_detector_instances = 1
+    ewma_detector_cpu_request = "100m"
+    ewma_detector_cpu_limit = "1000m"
+    ewma_detector_memory_request = "250"
+    ewma_detector_memory_limit = "250"
+    ewma_detector_jvm_memory_limit = "200"
+    ewma_detector_environment_overrides = ""
+
   }
```

Comment:
It looks like you didn't merge with my previous commit, and so are now adding these obsolete configs back to the project. :)